src/api/AuthServer/AuthServer/Services/Interfaces/IJwtService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace AuthServer.Services.Interfaces
+{
+    public interface IJwtService
+    {
+        string GenerateJwtToken(string userId, string userName, string email);
```

Comment:
Everything here is related to user, so we can leave a single argument of `ApplicationUser` type and take all the needed data from it